pkg/planner/core/planbuilder.go

```diff
@@ -2355,7 +2358,13 @@ func getModifiedIndexesInfoForAnalyze(
 	independentIdxsInfo = make([]*model.IndexInfo, 0)
 	specialGlobalIdxsInfo = make([]*model.IndexInfo, 0)
 	for _, originIdx := range tblInfo.Indices {
-		if originIdx.State != model.StatePublic {
+		// create index with analyze:
+		// * we analyze the reorg state index.
+		// change column with covered index analyze:
+		// * we analyze the public state index.
```

Comment:
Could you please explain this comment a bit more? What do you mean by ‘covered index analyze’ here? I thought that for lossy DDL, it should behave the same as creating a new index.